jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt

```diff
@@ -106,6 +107,10 @@ class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactor
         }
     }
 
+    private fun checkImageDebugger() {
+        imageDebugger() ?: throw RuntimeConfigurationError(message("lambda.image.missing_debugger", rawImageDebugger().toString()))
```

Comment:
`rawImageDebugger` returns String, do we need the toString()

Reply:
`rawImageDebugger` is nullable so we need the `toString` (or we could do `?: ""` but if a user reports it says `null` we'll know what actually happened)

Reply:
Does null not passed to message() print "null"?

Reply:
Nope, requires a string